### PR TITLE
Add Jupyter Idle server extensions recipes

### PR DESCRIPTION
**Pull request by rahrad123**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by rahrad123** [comment]
I agree to be listed as the maintainer

**Comment by rahrad123** [comment]
@conda-forge/help-python - the PR is ready for review. Thank you!

**Comment by rahrad123** [comment]
@conda-forge-admin, please ping conda-forge/help-python

**Comment by dmlause** [comment]
I also agree to be a maintainer. @conda-forge-manager - please let us know of any issues asap, thank you!

**Comment by rahrad123** [comment]
@conda-forge/help-python - the PR is ready for review. Thank you!

**Comment by rahrad123** [comment]
@conda-forge-admin, please ping conda-forge/help-python

**Comment by rahrad123** [comment]
@conda-forge-admin, please ping help-python

File: recipes/jupyter-idle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_idle-{{ version }}.tar.gz
+  sha256: cf8992dfaff47f6190d75f1f46ab99b5ed8f82543ac4d70013a3c88cd1159e7e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/jupyter-idle/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

**Comment by rahrad123** [comment]
@ocefpaf Thank you for the review. I have made the suggested change. Could you please take a look?

**Pull request merged**